vault/expiration.go

```diff
@@ -1056,7 +1066,7 @@ func (m *ExpirationManager) revokeEntry(le *leaseEntry) error {
 }
 
 // renewEntry is used to attempt renew of an internal entry
-func (m *ExpirationManager) renewEntry(le *leaseEntry) (*logical.Response, error) {
+func (m *ExpirationManager) renewEntry(le *leaseEntry, estimatedTTL time.Duration) (*logical.Response, error) {
```

Comment:
This isn't being used... is it just not implemented yet?

Reply:
Fixed in https://github.com/hashicorp/vault/pull/4230/commits/65e9e1b25b08f66c4dbe53144414f02586c61b70